training/cogvideox_text_to_video_lora.py

```diff
@@ -778,7 +778,7 @@ def load_model_hook(models, input_dir):
             last_lr = lr_scheduler.get_last_lr()[0] if lr_scheduler is not None else args.learning_rate
             logs = {"loss": loss.detach().item(), "lr": last_lr}
             # gradnorm + deepspeed: https://github.com/microsoft/DeepSpeed/issues/4555
-            if accelerator.distributed_type != DistributedType.DEEPSPEED:
+            if accelerator.sync_gradients and accelerator.distributed_type != DistributedType.DEEPSPEED:
```

Comment:
Should the OFS related changes be propagated here too?

Reply:
OFS only exists in image to video